server/server.go

```diff
@@ -127,6 +129,22 @@ func NewServer(n NewServerInput) (*Server, error) {
 		).ThenFunc(s.IndividualMessage),
 	).Methods(http.MethodGet)
 
+	s.Router.Handle("/edit",
+		alice.New(
+			s.CheckCookieExists,
```

Comment:
I don’t think we need to check if the session cookie exists here (or in the other /edit route). We should let people load this whether or not they have an existing inbox created.

Reply:
If I remove those checks, I get the following error on the edit POST action --

`CreateRouteFromInbox: failed to get sess var. Sess not of type sessions.Session actual type: *sessions.Session`

We need the session there, so not quite sure how to resolve?

Reply:
Argh. This middleware stuff is confusing and definitely needs a refactor. Just leave it for the moment  and I will work on fixing this stuff up.